CMakeLists_files.cmake

```diff
@@ -14,6 +14,7 @@ list( APPEND MAIN_SOURCE_FILES
 
 list (APPEND PUBLIC_HEADER_FILES
         opm/output/OutputWriter.hpp
+        opm/output/Wells.hpp
```

Comment:
You're absolutely right - the test program is in the cmake file on my machine, but I manage to press 'no' when adding it to the commit.